package.json

```
@@ -1,7 +1,16 @@
 {
   "name": "gulp-inject",
-  "version": "0.0.0-development",
+  "version": "4.3.2",
```

```suggestion
  "version": "0.0.0-development",
```

Revert this change, `semantic-release` handles the versioning without this